### ISSUE TITLE
Send and receive messages programmably via an API?

### ISSUE THREAD

**Issue by procedural** [Speek-App/Speek#5]
Since there are no servers and everything, can one send and receive messages via an API, programmably?

**Comment by procedural** [comment]
This looks like it, maybe?
https://github.com/Speek-App/Speek/blob/main/src/libtego/include/tego/tego.h

**Comment by Speek-App** [comment]
Thanks for opening an issue and the great feature request!
It is currently not possible to programmably send and receive messages. Though, we are planning to implement that feature soon, as it might be especially helpful with groups.

**Comment by BeholdersEye** [comment]
Even if you don't provide an API, a command line client would be very useful.

**Comment by Speek-App** [comment]
This feature is currently under development under the [feature/ipc-cli](/Speek-App/Speek/tree/feature/ipc-cli) branch, and once it is complete, it will be merged into the main codebase.

Please let us know if you have any special requests for us to take into consideration while we work on this.
